gripperShelf_4p SetGripperShelf
robotToolInfo Setrobottool
---
bool is_success
string info
